src/store/cache/user/graphHooks.ts

```diff
@@ -0,0 +1,457 @@
+      ? {
+          amount: new BN(user.pendingDecreaseStake.decreaseAmount),
+          lockupExpiryBlock: parseInt(user.pendingDecreaseStake.expiryBlock)
+        }
+      : { amount: new BN(0), lockupExpiryBlock: 0 }
+  }
+}
+
+// Async function to get
+function populateUsers(
+  users: FullUser[],
+  setStatus?: (status: Status) => void
+): ThunkAction<void, AppState, Audius, Action<string>> {
+  return async (dispatch, getState, aud) => {
+    try {
+      if (setStatus) setStatus(Status.Loading)
+      else dispatch(setLoading())
+      const formattedUsers = await Promise.all(
+        users.map(user => formatUser(aud, user))
+      )
+      dispatch(
+        setUsers({
+          users: formattedUsers.reduce(
+            (users: { [id: string]: User | Operator }, user) => {
+              users[user.wallet] = user
+              return users
+            },
+            {}
+          ),
+          ...(setStatus ? {} : { status: Status.Success })
+        })
+      )
+      if (setStatus) setStatus(Status.Success)
+    } catch (err) {
+      console.log({ err })
+    }
+  }
+}
+
+const GET_USERS = gql`
+  query users($where: User_filter!, $orderBy: User_orderBy!) {
+    users(where: $where, orderBy: $orderBy, orderDirection: desc) {
+      id
+      balance
+      totalClaimableAmount
+      stakeAmount
+      delegationSentAmount
+      delegationReceivedAmount
+      claimableStakeAmount
+      claimableDelegationSentAmount
+      claimableDelegationReceivedAmount
+      deployerCut
+      minAccountStake
+      maxAccountStake
+      validBounds
+
+      # Get the delegations received
+      delegateFrom(
+        orderBy: claimableAmount
+        orderDirection: desc
+        where: { amount_gt: 0 }
+      ) {
+        claimableAmount
+        amount
+        fromUser {
+          id
+        }
+      }
+
+      # Get the delegations sent
+      delegateTo(
+        orderBy: claimableAmount
+        orderDirection: desc
+        where: { amount_gt: 0 }
+      ) {
+        claimableAmount
+        amount
+        toUser {
+          id
+        }
+      }
+
+      # Get the pending actions
+      pendingDecreaseStake {
+        id
+        expiryBlock
+        decreaseAmount
+      }
+      pendingUndelegateStake {
+        id
+        amount
+        expiryBlock
+        serviceProvider {
+          id
+        }
+      }
+      pendingRemoveDelegator {
+        id
+        expiryBlock
+        delegator {
+          id
+        }
+      }
+      pendingUpdateDeployerCut {
+        id
+        expiryBlock
+        updatedCut
+      }
+
+      # Fetch the user's services
+      services(where: { isRegistered: true }) {
+        type {
+          id
+        }
+        spId
+        endpoint
+        delegateOwnerWallet
+      }
+
+      votes {
+        id
+        vote
+        magnitude
+        updatedBlockNumber
+        proposal {
+          id
+        }
+      }
+    }
+  }
+`
+
+interface FullUser {
```

Comment:
same with this? maybe put a hooks.ts, types.ts, and formatter.ts into  store/cach/user/graph